client/src/app/core/repositories/base-is-agenda-item-and-list-of-speakers-content-object-repository.ts

```diff
@@ -63,7 +63,13 @@ export abstract class BaseIsAgendaItemAndListOfSpeakersContentObjectRepository<
     }
 
     public getAgendaSlideTitle(titleInformation: T): string {
-        const numberPrefix = titleInformation.agenda_item_number() ? `${titleInformation.agenda_item_number()} · ` : '';
+        // TODO: Sometimes this is a function and sometime a property
+        const itemNumber =
+                typeof titleInformation.agenda_item_number === 'function'
+                    ? titleInformation.agenda_item_number()
+                    : titleInformation.agenda_item_number;
+
+        const numberPrefix = itemNumber ? `${itemNumber} · ` : '';
```

Comment:
@FinnStutzenstein Seems like something is wrong with the settings of `agenda_item_number`.
For normal models it is a function, for the `ListOfSpeakers` it might be a simple property.
I am working around that now. Any idea why that happens?

Reply:
Hi, since Github marks this as outdated: is it still relevant? If so, please message me, so I can check it this weekend, so it is done for the release at monday.

Reply:
yes it's relevant.